<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncodeDialog</class>
 <widget class="QDialog" name="EncodeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Encode video</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resources/vsedit.qrc">
    <normaloff>:/film_save.png</normaloff>:/film_save.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="VSEditorLog" name="feedbackTextEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="encodingPresetLabel">
       <property name="text">
        <string>Preset:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="encodingPresetComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::NoInsert</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="encodingPresetSaveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="encodingPresetDeleteButton">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Header:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="headerTypeComboBox">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Executable:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="executablePathEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="executableBrowseButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../resources/vsedit.qrc">
         <normaloff>:/folder.png</normaloff>:/folder.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Arguments (newlines will be replaced with spaces):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="argumentsHelpButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../resources/vsedit.qrc">
         <normaloff>:/information.png</normaloff>:/information.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="argumentsTextEdit">
     <property name="plainText">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="metricsEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="processingProgressBar">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="format">
      <string>%v / %m</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="framesLabel">
       <property name="text">
        <string>Frames:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="fromFrameSpinBox"/>
     </item>
     <item>
      <widget class="QLabel" name="toLabel">
       <property name="text">
        <string>to</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="toFrameSpinBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="wholeVideoButton">
       <property name="text">
        <string>Whole video</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="startEncodeButton">
       <property name="text">
        <string>Start</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pauseEncodeButton">
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="abortEncodeButton">
       <property name="text">
        <string>Abort</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VSEditorLog</class>
   <extends>QTextEdit</extends>
   <header>../../../common-src/log/vs_editor_log.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../resources/vsedit.qrc"/>
 </resources>
 <connections/>
</ui>
